hw/vendor/pulp_platform_fpnew.core

```diff
@@ -0,0 +1,48 @@
+CAPI=2:
+
+# Copyright 2021 OpenHW Group
+# Solderpad Hardware License, Version 2.1, see LICENSE.md for details.
+# SPDX-License-Identifier: Apache-2.0 WITH SHL-2.1
+
+name: "pulp-platform.org::fpnew"
+description: "PULP Platform RISC-V FPU"
+
+filesets:
+  files_rtl:
```

Comment:
I think this should be added here:
~~~
    depend:
    - pulp-platform.org::common_cells
~~~